src/metabase/server/middleware/session.clj

```diff
@@ -51,6 +51,15 @@
   [response]
   (reduce clear-cookie (wrap-body-if-needed response) [metabase-session-cookie metabase-embedded-session-cookie]))
 
+(defn- use-permanent-cookies?
+  "Check if we should use permanent cookies for a given request, which are not cleared when a browser sesion ends."
+  [request]
+  (if (config/config-bool :mb-session-cookies)
+    ;; Disallow permanent cookies if MB_SESSION_COOKIES is set
+    false
+    ;; Otherwise check whether the user selected "remember me" during login
+    (:remember (:body request))))
```

Comment:
You can do `(get-in request [:body :remember])` here, or even `(-> request :body :remember)` -- I think those are a little cleaner, especially if you have > 2 keys. Not worth changing here tho unless you feel like it.